libr/asm/arch/pyc/opcode.c

```diff
@@ -0,0 +1,266 @@
+	if(!strcmp("v3.6.9rc1", version)) return opcode_36();
+	if(!strcmp("v3.7.0", version))    return opcode_37();
+	if(!strcmp("v3.7.0a1", version))  return opcode_37();
+	if(!strcmp("v3.7.0a2", version))  return opcode_37();
+	if(!strcmp("v3.7.0a3", version))  return opcode_37();
+	if(!strcmp("v3.7.0a4", version))  return opcode_37();
+	if(!strcmp("v3.7.0b1", version))  return opcode_37();
+	if(!strcmp("v3.7.0b2", version))  return opcode_37();
+	if(!strcmp("v3.7.0b3", version))  return opcode_37();
+	if(!strcmp("v3.7.0b4", version))  return opcode_37();
+	if(!strcmp("v3.7.0b5", version))  return opcode_37();
+	if(!strcmp("v3.7.0rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.1", version))    return opcode_37();
+	if(!strcmp("v3.7.1rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.1rc2", version)) return opcode_37();
+	if(!strcmp("v3.7.2", version))    return opcode_37();
+	if(!strcmp("v3.7.2rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.3", version))    return opcode_37();
+	if(!strcmp("v3.7.3rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.4", version))    return opcode_37();
+	if(!strcmp("v3.7.4rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.4rc2", version)) return opcode_37();
+	if(!strcmp("v3.7.5", version))    return opcode_37();
+	if(!strcmp("v3.7.5rc1", version)) return opcode_37();
+	if(!strcmp("v3.7.6", version))    return opcode_37();
+	if(!strcmp("v3.7.6rc1", version)) return opcode_37();
+	if(!strcmp("v3.8.0", version))    return opcode_38();
+	if(!strcmp("v3.8.0a1", version))  return opcode_38();
+	if(!strcmp("v3.8.0a2", version))  return opcode_38();
+	if(!strcmp("v3.8.0a3", version))  return opcode_38();
+	if(!strcmp("v3.8.0a4", version))  return opcode_38();
+	if(!strcmp("v3.8.0b1", version))  return opcode_38();
+	if(!strcmp("v3.8.0b2", version))  return opcode_38();
+	if(!strcmp("v3.8.0b3", version))  return opcode_38();
+	if(!strcmp("v3.8.0b4", version))  return opcode_38();
+	if(!strcmp("v3.8.0rc1", version)) return opcode_38();
+	if(!strcmp("v3.8.1", version))    return opcode_38();
+	if(!strcmp("v3.8.1rc1", version)) return opcode_38();
+	if(!strcmp("v3.9.0a1", version))  return opcode_39();
+	if(!strcmp("v3.9.0a2", version))  return opcode_39();
+	if(!strcmp("v3.9.0a3", version))  return opcode_39();
+	return 0; // No match version
+}
+
+pyc_opcodes *new_pyc_opcodes() {
+	pyc_opcodes *ret = R_NEW0 (pyc_opcodes);
+	if (!ret)
+		return NULL;
+	ret->have_argument = 90;
+	ret->opcodes = malloc (sizeof(pyc_opcode_object) * 256);
+	if (!ret->opcodes) {
+		R_FREE(ret);
+		return NULL;
+	}
+	{
+	ut16 i = 0;
+	for(i = 0; i < 256; ++i) {
+		ret->opcodes[i].op_name = malloc (sizeof(char) * 6);
+		if (!ret->opcodes[i].op_name)
+			break;
+		snprintf (ret->opcodes[i].op_name, 6, "<%u>", i);
+		ret->opcodes[i].type = 0;
+		ret->opcodes[i].op_code = i;
+		ret->opcodes[i].op_push = 0;
+		ret->opcodes[i].op_pop = 0;
+	}
+	if (i != 256) {
+		for (ut8 j = 0; j < i; ++j)
```

Comment:
missing braces and unnecssary preincrement

Reply:
instead of putting it in here, just move this if (i!= 256) logic inside the break block